index.html

```diff
@@ -1,7 +1,7 @@
 ---
 title: Meshery | The Service Mesh Manager
 description: Lifecycle, performance, and configuration management across any service mesh. Confidently operate service meshes like Istio, Linkerd, Envoy, Citrix, Octarine, App Mesh, Consul, Kuma, Traefik Mesh, Tanzu, NGINX, and Open Service Mesh. Patterns and best practices of service mesh operation.
-image: images/logos/meshery-logo-light-text-pad.png
+image: https://raw.githubusercontent.com/meshery/meshery.io/master/images/logos/meshery-logo-tag-dark-text-side.png
```

Comment:
```suggestion
image: images/logos/meshery-logo-tag-dark-text-side.png
```
You could just import the pic like this, right?

Reply:
@warunicorn19 Yes, I have made the changes as suggested.